xenserver/xenserver-pxe.erb

```diff
@@ -0,0 +1,5 @@
+default xenserver
+label xenserver
+kernel mboot.c32
+append boot/XenServer-6.5-x86_64-xen.gz dom0_max_vcpus=16 dom0_mem=752M com1=115200,8n1 console=com1,vga --- boot/XenServer-6.5-x86_64-vmlinuz xencons=hvc console=hvc0 console=tty0 answerfile=<%= foreman_url('provision')%> install --- boot/XenServer-6.5-x86_64-initrd.img
```

Comment:
I am not sure if that's a good idea. General work flow for this setup to work is.
- Copy boot files from iso to tftp boot folder
- Create an http share and extract iso to that share (required for installation)
- Use this templates in foreman

I am not sure including this in foreman is a good idea or not. If it becomes part of standard foreman, there should be some sort of wizard or something that takes in iso and creates http share and copies required files into tight places.